### DISCUSSION TITLE
群晖alist怎么升级到最新版本啊

### DISCUSSION THREAD

**Discussion by gf11021** [AlistGo/alist#3442] [General]
目前通过alist管理看到的版本是v2.6.3-0-gXXXXX，想升级到最新版本。请问各位大侠指点一下

**Comment by chenjuefei** [comment]
1.现在最新的版本V3 与 V2不兼容，所以你可能需要重新配置。不能直接更新。
2.V3的版本，可以更新。群晖的更新方式很简单。先下载最新的镜像"latest"。不能是特定的版本。然后停止容器，再右击容器选择“重置”，旧版docker可能叫“清除”。之后再启动容器就可以了。
